content/documentation/device-examples/JoystickAnalog.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Joystick Analog
+weight: 208
+tags: ["Joystick analog"]
+---
+### Description
+The [analog Joystick](https://github.com/Pi4J/pi4j-example-components/tree/Dev-Arcade/src/main/java/com/pi4j/example/components) (src/main/java/com/pi4j/example/components) is a template class, that you can use in your own Java-project.
+The template class is created for an analog joystick, for example KY-023, with two potentiometers one each for the X-axis and the Y-axis. Additionally a button can be attached to the class.
```

Comment:
try: for example the KY-023, which consists of two potentiometers, one for the X-axis and one for the Y-axis. 
last sentence should be clear in the javadoc in the class itself.